pkg/bro-backend/config.go

```diff
@@ -0,0 +1,42 @@
+package brobackend
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/threeal/bro/pkg/utils"
+)
+
+var CONFIG_FILENAME = "backend_config.json"
+
+type Config struct {
+	ListenAddr *string `json:"listen_addr"`
+}
+
+func (c *Config) Read() error {
+	return utils.ReadConfigFromFile(c, CONFIG_FILENAME)
+}
+
+func (c *Config) Write() error {
+	return utils.WriteConfigToFile(c, CONFIG_FILENAME)
+}
+
+func (c *Config) Init(rd io.Reader) error {
+	if c.ListenAddr == nil {
+		reader := bufio.NewReader(rd)
+		fmt.Print(color.HiBlackString("question"), " listen address ", color.HiGreenString("(:320)"), ": ")
+		text, err := reader.ReadString('\n')
+		if err != nil {
+			return err
+		}
+		if text == "\n" {
+			text = ":320"
+		}
+		text = strings.TrimSpace(text)
+		c.ListenAddr = &text
```

Comment:
i see this pattern is being used by `pkg/bro`, let's modify it so somehow we could write it like this:
```go
text, err == m.Foo("listen address", ...) // we filter the '\n' here
if err != nil {
  return err
}
if text == "" {
  text = ":320"
} 
c.ListenAddr = text
```